ambry-router/src/main/java/com.github.ambry.router/CoordinatorBackedRouter.java

```diff
@@ -402,6 +405,7 @@ protected void onQueue() {
    * Tracks metrics on dequeuing of operation.
    */
   private void onDequeue() {
+    router.metrics.totalOperationsInExecution.incrementAndGet();
```

Comment:
perhaps this can go in where `onDequeue()` is called, so that the increment and decrement are within the same method?